rpcs3/Emu/SysCalls/lv2/sys_spu.cpp

```diff
@@ -508,7 +508,8 @@ s32 sys_spu_thread_group_terminate(u32 id, s32 value)
 		}
 	}
 
-	if (group->state <= SPU_THREAD_GROUP_STATUS_INITIALIZED || group->state == SPU_THREAD_GROUP_STATUS_WAITING || group->state == SPU_THREAD_GROUP_STATUS_WAITING)
+	// TODO:: Syphurith: Thank @danilaml (first) and let us talk about this mistake/typo. Have you used the wrong constant for the third condition?
+	if ((group->state <= SPU_THREAD_GROUP_STATUS_INITIALIZED) || (group->state == SPU_THREAD_GROUP_STATUS_WAITING) || (group->state == SPU_THREAD_GROUP_STATUS_WAITING))
```

Comment:
@Nekotekina You're right, `>=` higher than `==` and these higher than `||`.
I just used braces to classify. If annoyed please tell me to remove those.